src/btree/node.ts

```diff
@@ -56,6 +62,25 @@ export class BTreeNode {
     return this.internalPointers.length + this.leafPointers.length;
   }
 
+  async readChildNode(index: number) {
```

Comment:
```suggestion
  async child(index: number) {
```

This can be called just `child`, it's cleaner :)